src/main/java/com/redhat/cloud/common/clowder/configsource/handlers/QuarkusRedisClowderPropertyHandler.java

```diff
@@ -0,0 +1,34 @@
+package com.redhat.cloud.common.clowder.configsource.handlers;
+
+import com.redhat.cloud.common.clowder.configsource.ClowderConfig;
+import com.redhat.cloud.common.clowder.configsource.ClowderConfigSource;
+
+public class QuarkusRedisClowderPropertyHandler extends ClowderPropertyHandler {
+    private static final String QUARKUS_REDIS = "quarkus.redis.";
+
+    public QuarkusRedisClowderPropertyHandler(ClowderConfig clowderConfig) {
+        super(clowderConfig);
+    }
+
+    @Override
+    public boolean handles(String property) {
+        return property.startsWith(QUARKUS_REDIS);
+    }
+
+    @Override
+    public String handle(String property, ClowderConfigSource configSource) {
+        if (clowderConfig.inMemoryDb == null) {
+            throw new IllegalStateException("No inMemoryDb section found");
+        }
```

Comment:
I'm not sure which one is better between throwing an exception or logging a warn message (see `QuarkusUnleashClowderPropertyHandler`). Let's throw for now and maybe revisit that later if needed.